contracts/VaultFactory.vy

```diff
@@ -238,14 +276,13 @@ def set_custom_protocol_fee_bps(vault: address, new_custom_protocol_fee: uint16)
     """
     assert msg.sender == self.governance, "not governance"
     assert new_custom_protocol_fee <= MAX_FEE_BPS, "fee too high"
-    assert self.default_protocol_fee_config.fee_recipient != empty(address), "no recipient"
-
-    self.custom_protocol_fee[vault] = new_custom_protocol_fee
+    assert self._unpack_fee_recipient(self.default_protocol_fee_data) != empty(address), "no recipient"
 
-    # If this is the first time a custom fee is set for this vault
-    # set the bool indicator so it returns the correct fee.
-    if not self.use_custom_protocol_fee[vault]:
-        self.use_custom_protocol_fee[vault] = True
+    self.custom_protocol_fee_data[vault] = self._pack_data(
+        empty(address), 
```

Comment:
~~shouldn't this be equal to: `self._unpack_fee_recipient(self.default_protocol_fee_data)` ?~~

edit: nvm, you have a better approach on line `178` by getting the value from the default config. thus, we don't have to worry about the recipient ever being out-of-sync between custom/default.